framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -173,6 +174,8 @@
     /** The line separator. */
     private static final String LINE_SEPARATOR = System.lineSeparator().intern();
 
+    private final boolean isJdkAsStub;
```

Comment:
Some comment for the intuition for this field would be good.

Reply:
I added a comment.